content/post/voyager-gateway-v0.6.1/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Announcing Voyager Gateway v0.6.1
+date: "2023-11-30"
+weight: 14
+authors:
+- Tasdidur Rahman
+tags:
+- cloud-native
+- gateway
+- kubernetes
+- voyager-gateway
+---
+
+We are pleased to announce the release of **Voyager Gateway v0.6.1**. This post lists all the major changes done in this release since the last release.
+
+### Custom Database Routes
+
+Voyager Gateway is built on the Envoyproxy gateway project which uses Envoyproxy under the hood. Envoyproxy supports protocol specific filters for databases like MySQL, PostgreSQL, MongoDB, Kafka and Redis. But k8s gateway api does not have a way to specify the database filters. We can expose databases via TCP connections but this restricts full potential of the database filters in Envoyproxy.
```

Comment:
"Envoyproxy gateway" should be replaced with **Envoy Gateway**. And, Envoyproxy should only be called **Envoy**